### PR TITLE
RESTEasy multipart scenario - enhanced version

### PR DESCRIPTION
**Pull request by rsvoboda**
RESTEasy multipart scenario

### CODE REVIEW

**Comment by rsvoboda** [comment]
@jsmrcka please take a look, based on your work, but the finalization polish ++ usage of `byte[]` for image/data part makes it easier to understand

**Comment by kshpak** [comment]
@rsvoboda I would prefer to go with a dedicated module for the multipart scenario.

File: 001-quarkus-getting-started/src/test/java/io/quarkus/qe/core/AsciiMultipartResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@QuarkusTest
+@DisabledOnNativeImage // @TestProfile works in JVM mode
+@TestProfile(AsciiMultipartResourceTest.AsciiTestProfile.class)
+public class AsciiMultipartResourceTest {
+
+    public static final String TEXT_WITH_DIACRITICS = "Přikrášlený žloťoučký kůň úpěl ďábelské ódy.";
+    public static final byte[] EXPECTED_ASCII_BYTES = {80, -17, -65, -67, -17, -65, -67, 105, 107, 114, -17, -65, -67, -17, -65, -67, -17, -65, -67, -17, -65, -67, 108, 101, 110, -17, -65, -67, -17, -65, -67, 32, -17, -65, -67, -17, -65, -67, 108, 111, -17, -65, -67, -17, -65, -67, 111, 117, -17, -65, -67, -17, -65, -67, 107, -17, -65, -67, -17, -65, -67, 32, 107, -17, -65, -67, -17, -65, -67, -17, -65, -67, -17, -65, -67, 32, -17, -65, -67, -17, -65, -67, 112, -17, -65, -67, -17, -65, -67, 108, 32, -17, -65, -67, -17, -65, -67, -17, -65, -67, -17, -65, -67, 98, 101, 108, 115, 107, -17, -65, -67, -17, -65, -67, 32, -17, -65, -67, -17, -65, -67, 100, 121, 46};
```
Comment (jsmrcka) [inline 001-quarkus-getting-started/src/test/java/io/quarkus/qe/core/AsciiMultipartResourceTest.java:25]: Couldn't we do something like this:
```
private static final String EXPECTED_ASCII_TEXT = new String(TEXT_WITH_DIACRITICS.getBytes(StandardCharsets.UTF_8), StandardCharsets.US_ASCII);
```
and later test for:
```
...
.body(not(equalTo(TEXT_WITH_DIACRITICS)))
.body(equalTo(EXPECTED_ASCII_TEXT));
```

**Comment by rsvoboda** [comment]
PR updated, comments reflected.

File: 001-quarkus-getting-started-with-jaxrs/pom.xml [hunk 2]
```diff
@@ -16,5 +16,9 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-resteasy</artifactId>
     </dependency>
+    <dependency>
```
Comment (kshpak) [inline 001-quarkus-getting-started-with-jaxrs/pom.xml:19]: Adjust please <artifactId> to _001-quarkus-getting-started-with-jaxrs_

**Review by kshpak — CHANGES_REQUESTED** [review]
Adjust please **artifactId** to _001-quarkus-getting-started-with-jaxrs_ in module's pom.xml

**Comment by rsvoboda** [comment]
@kshpak PR updated, thanks for noticing

**Review by kshpak — APPROVED** [review]

**Review by jsmrcka — APPROVED** [review]

**Pull request merged**